1/debian-10/rootfs/opt/bitnami/scripts/libpgbouncer.sh

```diff
@@ -189,7 +189,16 @@ pgbouncer_initialize() {
     info "Creating configuration file"
     # Create configuration
     if ! pgbouncer_is_file_external "pgbouncer.ini"; then
-        ini-file set --section "databases" --key "$PGBOUNCER_DATABASE" --value "host=$POSTGRESQL_HOST port=$POSTGRESQL_PORT dbname=$POSTGRESQL_DATABASE" "$PGBOUNCER_CONF_FILE"
+        # Build DB string based on what the user has configured
+        # Allow for wildcard db config
+        DATABASE_VALUE="host=$POSTGRESQL_HOST port=$POSTGRESQL_PORT"
+        if [[ "$PGBOUNCER_SET_USER" ]]; then
+            DATABASE_VALUE+=" user=$POSTGRESQL_USERNAME"
+        fi
+        if [[ "$PGBOUNCER_DATABASE" != "*" ]]; then
+            DATABASE_VALUE+=" dbname=$POSTGRESQL_DATABASE"
+        fi
+        ini-file set --section "databases" --key "$PGBOUNCER_DATABASE" --value "$DATABASE_VALUE" "$PGBOUNCER_CONF_FILE"
```

Comment:
`DATABASE_VALUE` is a local variable (I mean is only used in the `pgbouncer_initialize` function scope), therefore you should treat it like that. Also, use `is_boolean_yes` (an aux. function we have) to deal with `yes/no` and `true/false` env. vars:
```suggestion
        local database_value="host=$POSTGRESQL_HOST port=$POSTGRESQL_PORT"
        if is_boolean_yes "$PGBOUNCER_SET_USER"; then
            database_value+=" user=$POSTGRESQL_USERNAME"
        fi
        if [[ "$PGBOUNCER_DATABASE" != "*" ]]; then
            database_value+=" dbname=$POSTGRESQL_DATABASE"
        fi
        ini-file set --section "databases" --key "$PGBOUNCER_DATABASE" --value "$database_value" "$PGBOUNCER_CONF_FILE"
```